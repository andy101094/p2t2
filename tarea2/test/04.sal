principal: src/cadena.cpp:342: nodo* siguiente(localizador_t, cadena_t): Assertion `localizador_en_cadena(loc, cad)' failed.
1>#  prueba de insertar_segmento_despues, segmento_cadena, separar_segmento, subcadena.
2>Segmento insertado después de loc.
3>
4>
5>