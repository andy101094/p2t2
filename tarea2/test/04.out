1>#  prueba de insertar_segmento_despues, segmento_cadena, separar_segmento, subcadena.
2>Segmento insertado después de loc.
3>
4>
5>
6>Segmento insertado después de loc.
7>(0,a)(3,d)(4,e)
8>(3,d)
9>loc al inicio.
10>Segmento insertado después de loc.
11>(0,a)(3,d)(4,e)
12>Segmento insertado después de loc.
13>(0,a)(1,b)(2,c)(3,d)(4,e)
14>loc al final.
15>Segmento insertado después de loc.
16>(0,a)(1,b)(2,c)(3,d)(4,e)(5,f)(6,g)
17>(5,f)
18>(3,d)(4,e)(5,f)(6,g)
19>(5,f)(6,g)
20>(0,a)
21>(6,g)
22>(3,d)
23>(0,a)(1,b)(2,c)
24>(0,a)(1,b)(2,c)(3,d)(4,e)(5,f)(6,g)
25>(6,g)
26>(3,d)
27>(0,a)
28>(1,b)(2,c)(4,e)(5,f)
29>(1,b)(2,c)(4,e)(5,f)
30>No se cumplen las precondiciones.
31>
32>Fin.
